test/cql-pytest/conftest.py

```diff
@@ -91,7 +91,7 @@ def this_dc(cql):
 @pytest.fixture(scope="session")
 def test_keyspace_tablets(cql, this_dc, has_tablets):
     if not is_scylla(cql) or not has_tablets:
-        pytest.skip("tablet-specific test skipped")
+        yield None
```

Comment:
This means that any test that wants to directly use this fixture, has to check the return value, because the fixture will not auto-skip, as usual.

Reply:
Yes, but I checked that no test uses this fixture directly - I thought the intention was that you use test_keyspace with parameters.

Reply:
Yes, for now, this is not a problem. We can deal with this, when we add the first test using this fixture.